extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -189,6 +189,7 @@ class AmpLightbox extends AMP.BaseElement {
     });
 
     if (this.isScrollable_) {
+      this.element./*OK*/scrollTop = this.element./*OK*/scrollTop || 1;
```

Comment:
The lightbox component is initially not visible (`display: none`), when user click on the open button, `activate()` is called. `activate()` -> `initialize_()` -> `finalizeOpen_()` -> `mutateElement` (this set the `display` to empty to make the element visible) -> `mutateElement.then` (this is the place where we should set the `scrollTop`)